src/policies/base/Policy.h

```diff
@@ -1,11 +1,55 @@
 #ifndef POLICY_H
 #define POLICY_H
 
-class Policy
-{
+#include "Memento.h"
```

Comment:
Hey there is an import issue here can you just make sure to import "utils/Memento.h" not "Memento.h"